kratos/python/add_c2c_variables_to_python.cpp

```diff
@@ -40,6 +50,36 @@
 #include "utilities/timer.h"
 
 
+
+#ifdef KRATOS_REGISTER_IN_PYTHON_FLAG_IMPLEMENTATION
```

Comment:
ouch, what are u doing here?

Reply:
hmm didnt notice that part. I just copied the files from perfoce... It seems like something that has always been there in the perforce side. Anyway, I have run my c2c tests without that block and everything seems to be fine, so I will just delete that part.

Reply:
Maybe @jrubiogonzalez knows about this?

Reply:
All that define block is no longer necessary (everything until `namespace kratos`)